community/basic_examples/hw_aware_synthesis.ipynb

```diff
@@ -0,0 +1,601 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    star_connectivity = [[0, i] for i in range(1, max_width-1)]
</pre><p>It will be good to print the connectivities you've created</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Sorry I was not clear enough, I meant plotting it as a graph (using e.g. matplotlib)</p>